src/app/pages/authenticationPage/AuthenticationPage.tsx

```diff
@@ -28,12 +28,7 @@ const AuthenticationPage = () => {
   return (
```

Comment:
I believe this Page and associated scss file can be deleted, since it's not used anywhere.
Please double check if everything works afterwards